chapter15/qli/ex4.pl

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+use 5.010001;
+
+given ( $ARGV[0] ) {
+  when ( undef ) { print "Please input a number as cmd argument! \n"; }
```

Comment:
Isn't this condition sort of redundant, because the next one will check to see if its a valid number anyway?